src/schema.ts

```diff
@@ -0,0 +1,45 @@
+import { gql } from "apollo-server";
+
+export const typeDefs = gql`
```

Comment:
can we try splitting the schema in to smaller entries in their own files and unite them here?

Reply:
Good remark, I did it, thanks